packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -230,6 +230,8 @@ export type CreateChildSummarizerNodeFn = (summarizeInternal: SummarizeInternalF
 export interface IFluidDataStoreContext extends EventEmitter, Partial<IProvideFluidDataStoreRegistry> {
     readonly documentId: string;
     readonly id: string;
+    // True if the data store is LocalDataStoreContext.
```

Comment:
Could you please describe semantics, not implementation?
I.e. I, as developer of Data store runtime, interact with context interface, not particular implementation behind this interface. What is it that I need to know about this property to properly implement Data Store runtime?

Reply:
Also, would be good to explain why this property needs to be known to the user of the context.

Reply:
Added description